lib/beaker/dsl.rb

```diff
@@ -79,5 +80,10 @@ module DSL
     include Beaker::DSL::Helpers
     include Beaker::DSL::InstallUtils
     include Beaker::DSL::Patterns
+    include Beaker::DSL::PE
+
+    def self.register(helper_module)
```

Comment:
are we actually registering modules with this? I remember exploring this a while back, and found that the register method wasn't going to work out real well...although I can't remember specifics.